dts/bindings/spi/nordic,nrf-spi-common.yaml

```diff
@@ -18,6 +18,14 @@ properties:
       description: |
         Maximum data rate the SPI peripheral can be driven at, in Hz.
 
+    overrun-character:
+      type: int
+      default: 0xff
+      description: |
+        The overrun character is used when all bytes from the TX buffer are
```

Comment:
This makes sense.

Suggestion to provide the user more hints about how this corresponds to registers:

```
        The overrun character (ORC) is used when all bytes from the TX buffer are
```